examples/plot_compare_reduction.py

```diff
@@ -73,3 +90,29 @@
 plt.ylim((0, 1))
 plt.legend(loc='upper left')
 plt.show()
```

Comment:
With the current scheme, it will generate the figure right after the code snipped and before the second section. It looks fine to me.